srcpkgs/finviz/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'finviz'
+pkgname=finviz
+version=1.4.4
+revision=1
+build_style=python3-module
+depends="python3-BeautifulSoup4 python3-tqdm python3-tenacity python3-aiohttp python3-cssselect"
```

Comment:
seems to be missing some dependencies? https://github.com/mariostoev/finviz/blob/v1.4.4/setup.py#L15-L23

Reply:
> seems to be missing some dependencies? https://github.com/mariostoev/finviz/blob/v1.4.4/setup.py#L15-L23

Good catch. Come to think of it, for user_agent in particular, I don't see a void python3 package for that.

Reply:
then you'll want to package that too, in this same PR

Reply:
> > seems to be missing some dependencies? https://github.com/mariostoev/finviz/blob/v1.4.4/setup.py#L15-L23
> 
> Good catch. Come to think of it, for user_agent in particular, I don't see a void python3 package for that.

In fact, I guess I should make a python module package for user_agent ?

Reply:
yes

Reply:
> yes

Since then this package depends on the package I will build that isn't yet merged, do I need to do anything special beyond setting up that pull request and including it python3-user_agent in the depends here?

Reply:
From the error I'm getting now, it looks like python3-six is a hostmakedepends?

Reply:
> From the error I'm getting now, it looks like python3-six is a hostmakedepends?

Got that part fixed, but with the newest errors I'm a bit stumped. Perhaps it requires the packages listed in here?
https://github.com/lorien/user_agent/blob/master/requirements_dev.txt

Reply:
> > From the error I'm getting now, it looks like python3-six is a hostmakedepends?
> 
> Got that part fixed, but with the newest errors I'm a bit stumped. Perhaps it requires the packages listed in here? https://github.com/lorien/user_agent/blob/master/requirements_dev.txt

Okay never mind, was actually able to find a module called 'ua' https://github.com/tombulled/ua
https://pypi.org/project/ua/